modules/users/client/api/search-users.api.js

```diff
@@ -1,4 +1,4 @@
-import axios from 'axios';
+import axios from '@/config/lib/axios';
```

Comment:
We might need to have a separate high-level folder like this for the client since config is currently serverside only and if we start mixing these, separating concerns might get harder in future.

Reply:
Perhaps [`modules/core/client/config/`](https://github.com/Trustroots/trustroots/tree/327a47bcb4420afdd95b98ae787594939e301400/modules/core/client/config) or would that be mixing with angular files too much?

We could very well rename some Angular files with `.angular.` just to make it clearer.